pom.xml

```diff
@@ -407,25 +407,25 @@
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-core</artifactId>
-        <version>${dropwizard.version}</version>
+        <version>${shaded.dropwizard.version}</version>
       </dependency>
 
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-jvm</artifactId>
-        <version>${dropwizard.version}</version>
+        <version>${shaded.dropwizard.version}</version>
       </dependency>
 
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-jmx</artifactId>
-        <version>${dropwizard.version}</version>
+        <version>${shaded.dropwizard.version}</version>
       </dependency>
 
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-ganglia</artifactId>
-        <version>${dropwizard.ganglia.version}</version>
+        <version>${shaded.dropwizard.ganglia.version}</version>
       </dependency>
```

Comment:
Oops, we should remove all the dropwizard dependencies and change all the imports to org.apache.ratis.thirdparty.com.codahale.metrics.*

@codings-dan , I have reopened RATIS-1601.

Reply:
@szetszwo Thanks for pointing out the problem, I'm not very familiar with this part, so I missed some changes. I've made a new pull request, please take a look, thanks! see https://github.com/apache/ratis/pull/671